experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -380,9 +370,10 @@ export function makeOpGenerator(
 		// a passive client to compare against is convenient.
 		// TODO: If clients can join & leave mid-test, 'the last client' isn't stable with the natural
 		// array operations.
+
```

Comment:
I would go ahead and fix the above TODO as part of this PR.

The fuzz harness always creates a "summarizer" client which is readonly, so test authors generally don't need to worry about making one of their clients readonly for comparison.

This logic probably all goes away anyway given previous comments about client selection, just figured I'd call it out.

Reply:
removed `treeIndex` logic with built in fuzz harness logic